wsd/wopi/StorageConnectionManager.hpp

```diff
@@ -49,104 +49,11 @@ class StorageConnectionManager final
-            "storage.ssl.enable", COOLWSD::getConfigValue<bool>("ssl.enable", true));
-
-#if ENABLE_DEBUG
-        char* StorageSSLEnabled = getenv("STORAGE_SSL_ENABLE");
-        if (StorageSSLEnabled != NULL)
-        {
-            if (!strcasecmp(StorageSSLEnabled, "true"))
-                SSLEnabled = true;
-            else if (!strcasecmp(StorageSSLEnabled, "false"))
-                SSLEnabled = false;
-        }
-#endif
-
-        if (SSLEnabled)
-        {
-            sslClientParams.certificateFile = COOLWSD::getPathFromConfigWithFallback(
-                "storage.ssl.cert_file_path", "ssl.cert_file_path");
-            sslClientParams.privateKeyFile = COOLWSD::getPathFromConfigWithFallback(
-                "storage.ssl.key_file_path", "ssl.key_file_path");
-            sslClientParams.caLocation = COOLWSD::getPathFromConfigWithFallback(
-                "storage.ssl.ca_file_path", "ssl.ca_file_path");
-            sslClientParams.cipherList = COOLWSD::getPathFromConfigWithFallback(
-                "storage.ssl.cipher_list", "ssl.cipher_list");
-
-            sslClientParams.verificationMode =
-                (sslClientParams.caLocation.empty() ? Poco::Net::Context::VERIFY_NONE
-                                                    : Poco::Net::Context::VERIFY_STRICT);
-            sslClientParams.loadDefaultCAs = true;
-        }
-        else
-            sslClientParams.verificationMode = Poco::Net::Context::VERIFY_NONE;
-
-        Poco::SharedPtr<Poco::Net::PrivateKeyPassphraseHandler> consoleClientHandler =
-            new Poco::Net::KeyConsoleHandler(false);
-        Poco::SharedPtr<Poco::Net::InvalidCertificateHandler> invalidClientCertHandler =
-            new Poco::Net::AcceptCertificateHandler(false);
-
-        Poco::Net::Context::Ptr sslClientContext =
-            new Poco::Net::Context(Poco::Net::Context::CLIENT_USE, sslClientParams);
-        sslClientContext->disableProtocols(Poco::Net::Context::Protocols::PROTO_SSLV2 |
-                                           Poco::Net::Context::Protocols::PROTO_SSLV3 |
-                                           Poco::Net::Context::Protocols::PROTO_TLSV1);
-        Poco::Net::SSLManager::instance().initializeClient(
-            consoleClientHandler, invalidClientCertHandler, sslClientContext);
-
-        // Initialize our client SSL context.
-        ssl::Manager::initializeClientContext(
-            sslClientParams.certificateFile, sslClientParams.privateKeyFile,
-            sslClientParams.caLocation, sslClientParams.cipherList,
-            sslClientParams.caLocation.empty() ? ssl::CertificateVerification::Disabled
-                                               : ssl::CertificateVerification::Required);
-        if (!ssl::Manager::isClientContextInitialized())
-            LOG_ERR("Failed to initialize Client SSL.");
-        else
-            LOG_INF("Initialized Client SSL.");
-#endif
-#else
-        FilesystemEnabled = true;
-#endif
-    }
-#endif
-
```

Comment:
@Ashod .. so you moved this up to the C++ implementation? This removed code was excluded anyways .. so some 'hidden' initialize() implementation used before?

Reply:
Yes, now it's enabled and replaced by the code moved (which was more recent).